docs/start/getting-started/fragments/vue/setup.md

```diff
@@ -230,7 +230,14 @@ yarn add aws-amplify @aws-amplify/ui-components
 </amplify-block>
 </amplify-block-switcher>
 
-The `@aws-amplify/ui-vue` package is a set of Vue-specific UI components that make it easy to integrate functionality like end-to-end authentication flows.
+<amplify-callout>
+
```

Comment:
Note that this empty line is necessary or the markdown won't be correctly parsed.